content/rs/administering/cluster-operations/updating-certificates.md

```diff
@@ -5,13 +5,13 @@ weight: $weight
 alwaysopen: false
 categories: ["RS"]
 ---
-Redis Enterprise Software (RS) uses self-signed certificates to encrypt
-the following traffic:
+Redis Enterprise Software (RS) uses self-signed certificates in its out of the box configuration to ensure the product is secure by default. This self-signed certificate is used to establish encryption-in-transit for the following traffic:
 
-- Management UI
-- REST API
-- Connections between clients and the database endpoint
+- RS Management User Interface (UI)
```

Comment:
@IAmATeaPot418 I don't think we need the RS at the beginning because it all belongs to RS

Reply:
Will resubmit to remove.